lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -12,19 +12,30 @@ module Overcommit::Hook::PreCommit
   #     enabled: true
   #     command: ['npm', 'run', 'lint', '--', '-f', 'compact']
   #
+  # If ESLint returns some errors, you can bindly pass by
+  # setting `bindly_pass` option to true.
+  #
   # Note: This hook supports only compact format.
   #
   # @see http://eslint.org/
   class EsLint < Base
     def run
       result = execute(command, args: applicable_files)
       output = result.stdout.chomp
+      messages = output.split("\n").grep(/Warning|Error/)
+
+      # Fail for issues relative to the tool used.
+      if !config['bindly_pass'] && messages.empty? && !result.success?
+        $stderr.puts result.stderrs
```

Comment:
Using `$stderr.puts` will break the formatting of the hook run. Output should be returned along with the `return` line below, e.g.:

```ruby
return [:fail, result.stderr]
```

Also, `stderrs` is a typo (should be `stderr`).